### PR TITLE
Portfolio Finished

### PR DESCRIPTION
**Pull request by LizanshMathur**
I've added a contact form and footer with social media links to my portfolio using Flexbox in HTML and CSS. Here's a summary of what I've done:

Created a new section for the contact form and added inputs for name, email, and message 😋
Used Flexbox to style the form and ensure it looks good on all screen sizes 😊
Added a footer with links to my social media profiles, also using Flexbox to style it🎉
Made sure the HTML and CSS is valid and accessible 😊

Coding partner @alizaamir123 😁

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -213,5 +213,60 @@ <h3>Skills</h3>
...
+                <p>Your email</p>
+                <label for="email">
+                    <input type="email" name="email" placeholder="Enter your email" required>
+                </label>
+              </div>
+              <div class="name-label">
+                  <p> Your message</p>
+                    <label for="Message">
+                        <textarea name="Message" id="message" placeholder="Enter your text here..." cols="30" rows="10" >
+                        </textarea>
```
Comment (Ibizugbe) [inline index.html:240]: Nice job integrating `formspre` with your form👍. Just as required in this milestone, validation should be added to the name input field that stops it from accepting more than 30 characters😉. kindly add that to your form👍.

image

Take note from the image above that you are also required to add a validation to your text field that stops a user from submitting an empty form or submitting the form if that field is empty😉 Kindly fix that👍.

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @LizanshMathur 👋👋👋, 

tryagain


Good job so far!👍.
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] Linters are passing👍.
- [x] Github flow have been used👍.
- [x] Form is aligned with `flexbox`👍.


## Required Changes ♻️

- [ ] Nice job on documenting your project👍. Currently, your readme has two `portfolio` heading😉. I think it was a mistake but you should kindly make adjustment to it👍 also, on the key features section, there's a text that looks like a template text, kindly see the image below and take it away👍.
image

image

- [ ] Kindly add the button interactions as given in the design as we are required to stick to the design in this milestone👍.
image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -213,5 +213,60 @@ <h3>Skills</h3>
...
+    <section class="contact">
+        <img class="logo" src="./img/image_geometry_1.png" alt="Logo">
+        <div class="contact-form">
+        <h2 id="contact">Get Started</h2>
+        <p>I'm always interested in hearing
+            about new projects, so if you'd like
+             to chat please get in touch.</p>
+        <form action="https://formspree.io/f/xqkonpny" method="post">
+            <div class="name-label">
+                <p>Your name</p>
```
Comment (TadesseDev) [inline index.html:225]: - [Optional] Dear @LizanshMathur, Good adding the changes requested by the previous reviewer, I notice Your design is a bit different from a given design, As a frontend engineer we are expected to deliver a webpage identical to the given design. Please take a look into this 
 ### Your design 
image
 
### Figma design 
image

**Review by TadesseDev — APPROVED** [review]
### Hi @LizanshMathur 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**